### DISCUSSION TITLE
[Feature request] Ability to view toots in tiles arrangement

### DISCUSSION THREAD

**Discussion by StavrosKalognom** [mastodon/mastodon#27778] [General]
When searching toots by tag, It would be great to be able to choose to view the results in one column that takes the whole width of the screen, and the resulted posts to be presented in tiles arrangement (i.e. as a matrix inside this column). This is useful especially for desktop/laptop users. I think that mastodon did that once.

**Comment by shaedrich** [comment]
There is the **Advanced web interface** under `/settings/preferences/appearance`
![grafik](https://github.com/user-attachments/assets/6ce41008-3e7d-4401-8f61-04010f3533d8)
but that's probably not what you mean
